python/spacewalk/satellite_tools/reposync.py

```diff
@@ -146,7 +146,7 @@ def __init__(self, plug, dir_name):
 
         for s in (m.group(1) for m in re.finditer(r'(?i)<a href="(.+?)"', dir_html.decode())):
             if not (re.match(r'/', s) or re.search(r'\?', s) or re.search(r'\.\.', s) or re.match(r'[a-zA-Z]+:', s) or
-                    re.search(r'\.rpm$', s)):
+                    re.search(r'\.rpm$', s) or re.search(r'\.mirrorlist$', s) or re.search(r'^#$', s)):
```

Comment:
[nitpick] To me these look like the regex engine is not needed. The first use https://docs.python.org/3/library/stdtypes.html#str.endswith ant the second a plain comparison. Same for the existing `re.search(r'\.rpm$', s)`.